src/style.css

```
@@ -1 +1,73 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"); 
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+body {
+  margin: 0;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  margin-right: 20px;
+  list-style-type: none;
+}
+
+.nav__link {
+  position: relative;
+  display: block;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  color: #000;
+  text-decoration: none;
+  text-transform: uppercase;
+  padding: 23px 0;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.header_logo-img {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.header_logo-_link {
+  display: block;
+}
+
+.nav__link:active::after,
+.is-active::after {
```

```suggestion
.is-active::after {
```
is-active is enough)